samples/packages/pulumi-typescript/.nvmrc

```diff
@@ -0,0 +1 @@
+16
```

Comment:
Shouldn't need this file if the >=16 condition is already set in `package.json` like the other packages/projects

Reply:
deleted per other comment